src/screens/chats/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { PureComponent } from 'react';
+import { View, Text, Touchable } from '../../components';
+import { StyleSheet } from 'react-native'
+import { connect } from 'react-redux';
+import { setChat } from '../../store/chats/actions'
+
+
+class Chats extends PureComponent {
+    handleOpenChat = (chat) => {
+        const { onChatOpen } = this.props
+        onChatOpen(chat)
+    }
+
+    render() {
+        const { chats } = this.props
+        return (
+            <View style={styles.container}>
+                {
+                    chats.map(chat => (
+                        <View key={chat._id}>
+                            <Touchable
```

Comment:
Please, use auto formatter
`  <Touchable onPress={() => this.handleOpenChat(chat)}>`